src/main/java/de/tum/in/www1/artemis/domain/quiz/QuizExercise.java

```diff
@@ -412,6 +412,7 @@ public void undoUnallowedChanges(QuizExercise originalQuizExercise) {
         // reset unchangeable attributes: ( dueDate, releaseDate, question.points)
         this.setDueDate(originalQuizExercise.getDueDate());
         this.setReleaseDate(originalQuizExercise.getReleaseDate());
+        this.setStartDate(originalQuizExercise.getStartDate());
```

Comment:
Seems to be hack just to get start date working with QuizExercises. Like in my other comments I would suggest just using the property for programming exercises.

Reply:
Will be used for all exercises in the future

Reply:
See me comment from above, why do it in this PR already?

Reply:
If you really want to leave it like this, maybe add at least a comment `Currently only used for ....` and a ToDo to introduce to explain that this "hack" has to be changed once it is activated for quiz exercises.